modin/pandas/test/test_api.py

```diff
@@ -51,6 +51,7 @@ def test_top_level_api_equality():
         "base",
         "utils",
         "dataframe",
+        "groupby",
```

Comment:
Because there is no easy way to make a submodule _disappear_ from result of calling a `dir(module)`, and we need `modin.pandas.groupby` to be imported in the very beginning to RPyC to correctly make a netref class.